src/mscorlib/model.xml

```diff
@@ -8608,6 +8608,14 @@
       <Member Name="GetHashCode" />
       <Member Name="ToString" />
       <Member MemberType="Property" Name="Item1" />
+      <Member Status="ImplRoot" Name="System.Collections.IStructuralComparable.CompareTo(System.Object,System.Collections.IComparer)" />
```

Comment:
@tarekgh Thanks for the `modelgen` tool. I copied the bits that I think are useful here, but I don't really know how to validate.
In particular, I'm surprised that these pre-existing Tuple APIs weren't in the model.xml. Is adding them correct?